src/methods.rs

```diff
@@ -8,10 +9,10 @@ mod private {
     pub trait Sealed {}
 }
 
-pub trait RpcMethod: private::Sealed
+pub trait RpcMethod: private::Sealed + Debug
```

Comment:
Trait signatures typically only need to include information about what is useful to the defining crate.
The usefulness of this `Debug` bound is only realized outside the scope of the client as it's between the users of the client and `near-jsonrpc-primitives`. I'd prefer if the client didn't have to include this just because it has a unifying trait for all RPC methods, responses, and errors. Atm, we know all RPC methods, responses, and errors impl `Debug` but if we need a type system guarantee that it does, that would need to be implemented upstream in the nearcore crate.

Reply:
I see your point, but I don't see how I can change anything upstream (in nearcore crate) for the user of jsonrpc client to have a typesystem guarantee that I want. A user of jsonrpc client (such as workspaces) operates on instances that implement `RpcMethod` and its associated response/error types and, from what I can tell with my limited exposure to Rust, there is no way to "downcast" those instances to a specific type. So I am not sure what your suggestion is, could you elaborate plase?

Reply:
Well, I guess it depends on your use case. Are you doing type erasure? Storing something like `Box<dyn RpcHandlerResponse>` or `impl RpcHandlerResponse` and then trying to `println!("{:?}", response)`

Because otherwise, the types are always going to be known. And so far as the type itself implements `Debug`, there's no need to have this trait bound in the client.

Reply:
If you can show a snippet of what you'd like to do, I can tailor my comments to match.

Reply:
You were right, this change is indeed no necessary to achieve what I wanted. After playing with the code for a bit I found a way, thanks!

I will just merge this PR as a simple nearcore version bump then and will proceed to working directly on workspaces.